File: Algorithms/905. Sort Array By Parity.py
class Solution:
    def sortArrayByParity(self, A: List[int]) -> List[int]:
        ret = []
        for i in A:
            if i % 2 == 0:
                ret.append(i)
        
        for i in A:
            if i % 2:
                ret.append(i)
        
        return ret
        

File: Daily Challenges/2022/May/905. Sort Array By Parity.cpp
class Solution {
public:
    vector<int> sortArrayByParity(vector<int>& nums) {
        vector<int> result;
        
        for(int e : nums) {
            if(e % 2 == 0) {
                result.push_back(e);
            }
        }
        
        for(int e : nums) {
            if(e % 2) {
                result.push_back(e);
            }
        }
        
        return result;
    }
};
